src/vector/README.md

```diff
@@ -0,0 +1,42 @@
+# Vector
+
+Vector is a lightweight tool for building observability pipelines, built and maintained primarily by Datadog. Within UDS Core it is primarily used for log collection and shipping to destinations (like Loki and S3).
+
+## Switching from Promtail to Vector
+
+Within UDS Core we have made the decision to switch from Promtail (historically the log collector/shipper of choice) to Vector. The below contains primary motivating factors and impacts of this choice.
+
+### Motivations
+
+Promtail has historically been the tool of choice for log collection/shipping when using Loki. It provides a very lightweight layer to scrape logs from pods and hosts, label them with additional metadata, and ship them to Loki.
+
+One of the main issues that has arisen with Promtail is its limited output/export options. Promtail only supports sending logs to one or more Loki instances. A common requirement in production environments is to ship logs to a secondary destination for collection/analysis by security teams and SIEM tools. Promtail is currently listed as [feature complete](https://grafana.com/docs/loki/latest/send-data/promtail/) so there is no expectation that additional export functionality would be added.
+
+### Goals and Options
+
+In choosing an alternative to Promtail we have a few primary objectives:
+1. Chosen tool must be capable of gathering host and pod logs: This has been our primary usage of Promtail in the past - gathering pods logs and host logs (to include k8s audit logs, controlplane logs, etc).
```

Comment:
nit: Im partial to using bullets here if there isn't any sequence to how it should be read.

Reply:
Will make that change - might wait on more reviews to not burn CI on a small docs modification but taking note of this.